fem/bilinearform.hpp

```diff
@@ -438,6 +438,26 @@ class BilinearForm : public Matrix
    virtual const Operator *GetOutputRestriction() const
    { return GetRestriction(); }
 
+   /// @brief Compute serial RAP operator and store it in @a A as a SparseMatrix.
+   void SerialRAP(OperatorHandle &A)
+   {
+      MFEM_ASSERT(mat, "SerialRAP requires the SparseMatrix to be assembled.");
+      ConformingAssemble();
+      A.Reset(mat, false);
+   }
+
+   /** @brief Eliminate essential (Dirichlet) boundary conditions in the
+       SparseMatrix @a A
+
+       @param[in] ess_dofs indices of the degrees of freedom belonging to the
+                           essential boundary conditions.
+       @param[in] diag_policy policy for diagonal entries.
+       @param[in,out] A The SparseMatrix in which the boundary conditions are
+                        eliminated.  */
+   static void EliminateBC(const Array<int> &ess_dofs,
+                           DiagonalPolicy diag_policy,
+                           SparseMatrix &A);
```

Comment:
Would this maybe make sense as a member function of `SparseMatrix` instead (or free function)?

Reply:
Moved to `SparseMatrix`, and the parallel version is moved to `HypreParMatrix`.